packages/conversation/tests/rest/v1/templates-v2/templates-v2.steps.ts

```diff
@@ -0,0 +1,189 @@
+import { Conversation, ConversationService, SupportedConversationRegion, TemplatesV2Api } from '../../../../src';
+import { Given, Then, When } from '@cucumber/cucumber';
+import * as assert from 'assert';
+
+let templatesV2Api: TemplatesV2Api;
+let template: Conversation.V2Template;
+let templatesList: Conversation.V2ListTemplatesResponse;
+let translationsList: Conversation.V2ListTranslationsResponse;
+let deleteTemplateResponse: void;
+let currentVersion: number;
+
+Given('the Conversation service "TemplatesV2" is available', function () {
+  const conversationService = new ConversationService({
+    projectId: 'tinyfrog-jump-high-over-lilypadbasin',
+    keyId: 'keyId',
+    keySecret: 'keySecret',
+    authHostname: 'http://localhost:3011',
+    conversationTemplatesHostname: 'http://localhost:3015',
+    conversationRegion: SupportedConversationRegion.UNITED_STATES,
+  });
+  templatesV2Api = conversationService.templatesV2;
+});
+
+When('I send a request to create a conversation template with the V2 API', async () => {
+  template = await templatesV2Api.create({
+    createTemplateRequestBody: {
+      id: '01HVN010MG3B9N6X323JAFN59P',
+      default_translation: 'en-US',
+      description: 'Text template V2',
+      translations: [
+        {
+          language_code: 'en-US',
+          version: '3',
+          text_message: {
+            text: 'Hello ${name}. Text message template created with V2 API',
+          },
+          variables: [
+            {
+              key: 'name',
+              preview_value: 'Professor Jones',
+            },
+          ],
+        },
+      ],
+    },
+  });
+});
+
+Then('the conversation template V2 is created', () => {
+  assert.equal(template.id, '01HVN010MG3B9N6X323JAFN59P');
+  assert.equal(template.version, 1);
+  assert.equal(template.translations?.length, 1);
+  assert.equal(template.translations?.[0].version, '3');
+});
+
+When('I send a request to list the conversation templates with the V2 API', async () => {
+  templatesList = await templatesV2Api.list({});
+});
+
+Then('the response contains the list of conversation templates with the V2 structure', () => {
+  assert.equal(templatesList.templates?.length, 2);
+  for(const templateV2 of templatesList.templates!) {
+    assert.ok(templateV2.version);
+    let latestVersionCount = 0;
+    let otherVersionCount = 0;
+    const translations = templateV2.translations!;
+    for(const translation of translations) {
+      translation.version === 'latest' ? latestVersionCount++ : otherVersionCount++;
+    }
+    assert.equal(latestVersionCount, otherVersionCount);
```

Comment:
Not having yet managed templates, I'm onboarding about this test validation and discovering by reading the test node source itself: it is performing some business logic validation not described by the test content description: `the response contains the list of conversation templates with the V2 structure`
So when I'll write the Java e2e, I could be not fully aware about this test content validation.
Can we imagine having a test description ensuring we will validate properly this business logic ?

Reply:
Right, it deserves its own statement

Reply:
Fixed